flexmeasures/data/schemas/forecasting/pipeline.py

```diff
@@ -51,6 +51,7 @@ class ForecasterParametersSchema(Schema):
     probabilistic = fields.Bool(required=True)
     sensor_to_save = SensorIdField(required=False, allow_none=True)
     ensure_positive = fields.Bool(required=False, allow_none=True)
+    missing_threshold = fields.Float(required=False, allow_none=True, load_default=1.0)
```

Comment:
`allow_none` should be removed, I believe. I fear that it may allow specifying a None value explicitly, which our code might not deal with correctly. And in any case, the `load_default` should already kick in when the field is missing.